developers/utils/i18n.md

```diff
@@ -475,3 +475,17 @@ List<ThingType> thingTypes = thingTypeRegistry.getThingTypes(Locale.GERMAN);
 If one binding supports the German language and another does not, it might occur that the languages of the returned thing types are mixed.
 
 For Binding Info and ConfigDescription, the localized objects can be retrieved via the `BindingInfoRegistry` and the `ConfigDescriptionRegistry` in the same manner as described for Thing Types.
+
+## Managing translations
+
+Localization is managed with the cloud service [Crowdin](https://crowdin.com/) where translators can contribute translations.
+
+This is the process:
+
+- Texts in the original language (English) are provided (with the corresponding code) in pull requests.
+- When a pull request is merged, the texts will be automatically imported into Crowdin.
+- Texts can now be translated into multiple languages using Crowdin.
+- Once a day a pull request with updated translations is automatically created. Please note that all texts must be translated and approved in order to be included.
```

Comment:
```suggestion
- Once a day a pull request with updated translations is automatically created.
Please note that all texts must be translated and approved in order to be included.
```

Reply:
Thanks, I didn't know this would work inside an ordered list.